src/extensions/Statiq.Html/MirrorResources.cs

```diff
@@ -137,11 +137,14 @@ private async Task<string> DownloadAndReplaceAsync(string source, Dictionary<str
             {
                 source = $"http:{source}";
             }
-            if (!Uri.TryCreate(source, UriKind.Absolute, out Uri uri))
+
+            Uri.TryCreate(source, UriKind.Absolute, out Uri uri);
+            if (uri is null ||
```

Comment:
Is the out guaranteed to be null if the try fails or would it be better to check the `TryCreate` result here?

Reply:
I don't know why it wouldn't be null since it wouldn't be initialized.  It wouldn't hurt to check both though, if you want me to.